flytekit/tools/fast_registration.py

```diff
@@ -105,10 +103,14 @@ def download_distribution(additional_distribution: str, destination: str):
     :param Text additional_distribution:
     :param os.PathLike destination:
     """
-    file_access.get_data(additional_distribution, destination)
+    if not destination:
+        raise ValueError("Destination path is required to download distribution and it should be a directory")
```

Comment:
should we use https://docs.python.org/3/library/os.path.html#os.path.isdir and raise if it returns false?